src/python/nimbusml/internal/utils/entrypoints.py

```diff
@@ -466,16 +474,19 @@ def remove_multi_level_index(c):
                 return self._process_graph_run_results(out_data)
             elif self._data_output_format == DataOutputFormat.IDV:
                 output = BinaryDataStream(output_idvfilename)
-                return (output_modelfilename, output, out_metrics)
+                return (output_modelfilename, output, out_metrics, output_predictor_modelfilename)
             else:
-                return (output_modelfilename, out_data, out_metrics)
+                return (output_modelfilename, out_data, out_metrics, output_predictor_modelfilename)
         finally:
             if cv:
                 self._remove_temp_files()
             else:
                 if output_modelfilename:
                     # os.remove(output_modelfilename)
                     pass
+                if output_predictor_modelfilename:
```

Comment:
Should this and the one above it (`output_modelfilename`) be removed since they don't do anything?